src/libs/pois.js

```diff
@@ -13,15 +13,23 @@ export function toUrl(poi) {
 }
 
 export async function getInputValue(poi) {
-  if (poi.type === 'latlon' || !poi.name) {
+  if (poi.type === 'latlon' || !poi.address) {
     return await fetchAddressLabel(poi);
   }
-  return poi.getInputValue();
```

Comment:
This is what cause the regression on direction input fields, which is quite big IMO compared to the one on favorites.
I'm totally ok to eventually get rid of the `getInputValue` method on POI classes (I even think it can be removed in after this change), but always returning the address isn't the best solution. The address should be a fallback, but if the POI has a real name it should be displayed in priority.

Reply:
I don't said it's a regression, I though it was the expected behavior, to get only full addresses in inputs according to the ticket.
Should we display `Musée d'Orsay, Paris, France` in this case ?

Reply:
This is resolved in the latest commit (https://github.com/QwantResearch/erdapfel/pull/676/commits/c3ea2e8ccf1998bef5b6b9c95c57a1db2359065c) Is it ok ?

Reply:
> I don't said it's a regression

But it is :) "1 rue de la légion d'Honneur, Paris" makes no sense to much people, whereas "Musée d'Orsay" does.
I don't know which ticket you're referring to, there is no mention of directions in 1531, it's only about the 2nd line on suggestions.
I'd say for we just display `Musée d'Orsay` in this case, i.e. the POI name if it's available, like currently.

Reply:
Okay, it makes sense. I updated the PR, it should be more convenient to read and user these `getStreetAddress` and `getNameAddress` now :). Plus, we don't need `getInputValue` anymore!